pkg/forwarder/forwarder.go

```diff
@@ -124,9 +124,10 @@ const (
 )
 
 const (
-	apiHTTPHeaderKey       = "DD-Api-Key"
-	versionHTTPHeaderKey   = "DD-Agent-Version"
-	useragentHTTPHeaderKey = "User-Agent"
+	apiHTTPHeaderKey          = "DD-Api-Key"
+	versionHTTPHeaderKey      = "DD-Agent-Version"
+	useragentHTTPHeaderKey    = "User-Agent"
+	arbitraryTagHTTPHeaderKey = "Allow-Arbitrary-Tag-Value"
```

Comment:
I'm guessing this was discussed with intake teams, but any thoughts on using a `DD-` for this header? To reflect that it's DD-specific.

Reply:
I'm not sure, I can ask, but at this point I think it is unlikely to change. They might add an additional header though.